src/createGithubService.spec.ts

```diff
@@ -0,0 +1,57 @@
+import 'mocha';
+import {expect, use} from 'chai';
+import * as sinon from 'sinon';
+import * as sinonChai from 'sinon-chai';
+import * as Chance from 'chance';
+import createGithubService from './createGithubService';
+
+use(sinonChai);
```

Comment:
I think that if you create a `mocha-setup.ts` file and apply it there, it will apply it once for to all test files